src/_pytest/main.py

```diff
@@ -78,7 +78,7 @@ def pytest_addoption(parser: Parser) -> None:
     group._addoption(
         "--strict-config",
         action="store_true",
-        help="invalid ini keys for the `pytest` section of the configuration file raise errors.",
+        help="any warnings encountered while parsing the `pytest` section of the configuration file raise errors.",
```

Comment:
I think this should be reverted now?

Reply:
I don’t see why. This statement is valid. The one warning in the config will be raised as an error

Reply:
And more generally I think strict_config should apply to the entire config file parsing, for consistencies sake